sdk/tables/azure-data-tables/tests/test_table_cosmos.py

```diff
@@ -191,41 +191,3 @@ def test_delete_table_with_non_existing_table_fail_not_exist(self, tables_cosmos
         ts = TableServiceClient(self.account_url(tables_cosmos_account_name, "cosmos"), credential=tables_primary_cosmos_account_key)
         table_name = self._get_table_reference()
         ts.delete_table(table_name)
-
-
-class TestTableUnitTest(TableTestCase):
-    tables_cosmos_account_name = "fake_storage_account"
-    tables_primary_cosmos_account_key = "fakeXMZjnGsZGvd4bVr3Il5SeHA"
-    credential = AzureNamedKeyCredential(name=tables_cosmos_account_name, key=tables_primary_cosmos_account_key)
-
-    def test_create_table_invalid_name(self):
```

Comment:
Could we add these tests back?
For cosmos these should all be successful table names rather than errors.... but it would still be good to test.

Reply:
Will do